### PR TITLE
Fixes #7676 - order dashboard errata list by errata updated date.

### PR DESCRIPTION
**Pull request by waldenraines**
http://projects.theforeman.org/issues/7676

### CODE REVIEW

File: app/controllers/katello/dashboard_controller.rb [hunk 1]
```diff
@@ -42,9 +42,7 @@ def errata
       # but since a system cannot be registered to Library,
       # skip repos in Library
       systems = System.in_organization(current_organization).readable
-      errata = Erratum.applicable_to_systems(systems)
-
-      errata = errata.sort_by { |e| e.systems_applicable.in_organization(current_organization).readable.count }.reverse[0...quantity]
+      errata = Erratum.applicable_to_systems(systems).order(:updated)
```
Comment (jlsherrill) [inline app/controllers/katello/dashboard_controller.rb]: i think you'd need a limit(quantity) on there, right?

Comment (waldenraines) [inline app/controllers/katello/dashboard_controller.rb]: Ah, good catch.

**Comment by waldenraines** [comment]
@jlsherrill updated.

**Comment by jlsherrill** [comment]
looks good, but I think its sorted incorrectly, i think you need   .order('updated desc')

**Comment by waldenraines** [comment]
@jlsherrill updated.

**Comment by jlsherrill** [comment]
APJ

**Pull request merged**